application/jaxrs-mongodb-quarkus/src/test/java/webapp/tier/service/MongodbServiceTest.java

```diff
@@ -1,45 +1,93 @@
 package webapp.tier.service;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.when;
 
+import java.security.NoSuchAlgorithmException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.bson.Document;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoCursor;
+
+import io.quarkus.test.junit.QuarkusTest;
+import webapp.tier.bean.MsgBean;
 
+@QuarkusTest
 class MongodbServiceTest {
 
+	@Inject
+	MongodbService mongosvc;
+
+	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5923183)